app/views/spree/shared/_registration_header.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav class="breadcrumb-wrap d-flex flex-wrap align-items-center justify-content-between">
```

Comment:
Remove this breadcrumb code here and call spree breadcrumb partial instead. Please discuss with Umer.